apertium-eng.eng.dix

```diff
@@ -65069,133 +65079,412 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+<e lm="Savona">          <i>Savona</i><par n="Barcelona__np"/></e>
 <e lm="Savoy">           <i>Savoy</i><par n="Barcelona__np"/></e>
 <e lm="Saxony">          <i>Saxony</i><par n="Barcelona__np"/></e>
+<e lm="Sayama">          <i>Sayama</i><par n="Barcelona__np"/></e>
+<e lm="Sayhat">          <i>Sayhat</i><par n="Barcelona__np"/></e>
+<e lm="Sayyan">          <i>Sayyan</i><par n="Barcelona__np"/></e>
 <e lm="Scandinavia">     <i>Scandinavia</i><par n="Barcelona__np"/></e>
 <e lm="Scarborough">     <i>Scarborough</i><par n="Barcelona__np"/></e>
 <e lm="Scheldt">         <i>Scheldt</i><par n="Barcelona__np"/></e>
 <e lm="Schleswig">       <i>Schleswig</i><par n="Barcelona__np"/></e>
+<e lm="Schweizer-Reneke"><i>Schweizer-Reneke</i><par n="Barcelona__np"/></e>
+<e lm="Schwerte">        <i>Schwerte</i><par n="Barcelona__np"/></e>
 <e lm="Scotland">        <i>Scotland</i><par n="Barcelona__np"/></e>
+<e lm="Scottsdale">      <i>Scottsdale</i><par n="Barcelona__np"/></e>
+<e lm="Scranton">        <i>Scranton</i><par n="Barcelona__np"/></e>
 <e lm="Sealand">         <i>Sealand</i><par n="Barcelona__np"/></e>
 <e lm="Seattle">         <i>Seattle</i><par n="Barcelona__np"/></e>
+<e lm="Sector 3">        <i>Sector<b/>3</i><par n="Barcelona__np"/></e>
+<e lm="Segou">           <i>Segou</i><par n="Barcelona__np"/></e>
+<e lm="Segovia">         <i>Segovia</i><par n="Barcelona__np"/></e>
+<e lm="Seguela">         <i>Seguela</i><par n="Barcelona__np"/></e>
 <e lm="Seine">           <i>Seine</i><par n="Barcelona__np"/></e>
+<e lm="Sejong">          <i>Sejong</i><par n="Barcelona__np"/></e>
 <e lm="Selkirk">         <i>Selkirk</i><par n="Barcelona__np"/></e>
+<e lm="Semarang">        <i>Semarang</i><par n="Barcelona__np"/></e>
+<e lm="Semey">           <i>Semey</i><par n="Barcelona__np"/></e>
+<e lm="Sendhwa">         <i>Sendhwa</i><par n="Barcelona__np"/></e>
 <e lm="Senegal">         <i>Senegal</i><par n="Barcelona__np"/></e>
+<e lm="Sengkang">        <i>Sengkang</i><par n="Barcelona__np"/></e>
+<e lm="Seogwipo">        <i>Seogwipo</i><par n="Barcelona__np"/></e>
+<e lm="Seoni">           <i>Seoni</i><par n="Barcelona__np"/></e>
+<e lm="Seosan">          <i>Seosan</i><par n="Barcelona__np"/></e>
 <e lm="Seoul">           <i>Seoul</i><par n="Barcelona__np"/></e>
+<e lm="Sepang">          <i>Sepang</i><par n="Barcelona__np"/></e>
+<e lm="Seraing">         <i>Seraing</i><par n="Barcelona__np"/></e>
+<e lm="Serang">          <i>Serang</i><par n="Barcelona__np"/></e>
 <e lm="Serbia">          <i>Serbia</i><par n="Barcelona__np"/></e>
+<e lm="Sergiyev Posad">  <i>Sergiyev<b/>Posad</i><par n="Barcelona__np"/></e>
+<e lm="Seropedica">      <i>Seropedica</i><par n="Barcelona__np"/></e>
+<e lm="Serpukhov">       <i>Serpukhov</i><par n="Barcelona__np"/></e>
+<e lm="Serra Talhada">   <i>Serra<b/>Talhada</i><par n="Barcelona__np"/></e>
+<e lm="Sete Lagoas">     <i>Sete<b/>Lagoas</i><par n="Barcelona__np"/></e>
```

Comment:
Is this categorised right?  Check categories for place names that are compositionally plurals.

Reply:
i cant understand. can u please explain

Reply:
Yeah that is a city in Brazil, and everything i added in places name are cities! :)

Reply:
Does it matter that they're cities?  Note the others you added that are plurals too.

Reply:
So can i remove those are plurals?

Reply:
okay i got it now!